marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -90,6 +90,21 @@
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/{modelName}", method = RequestMethod.DELETE)
+    @Secured("ROLE_writeEntityModel")
+    public ResponseEntity<Void> deleteModel(@PathVariable String modelName) {
+        newService().deleteModel(modelName);
```

Comment:
I think it's important to document why we're not doing anything with indexes or protected paths here because we don't have a reliable way to identify the ones to delete and we only expect entity types to be deleted in a development environment where it's usually fine if some old indexes / protected paths hang around. I say this because when I looked at this, I immediately thought - what about indexes? - before remembering that we're taking the approach above. 6 months from now, we'll be glad we have a comment here explaining why.